src/style.css

```diff
@@ -1 +1,40 @@
 /* styles go here */
+fieldset {
+  margin-bottom: 20px;
+}
+
+input {
+  margin-bottom: 10px;
+}
+
+.termsAndCond {
+  margin-bottom: 0%;
+}
+
+.passwordReg {
+  margin-bottom: 0%;
+}
+
+.loveCatts {
+  margin-bottom: 0%;
+}
+
+.favColor {
+  margin-top: 10px;
+}
+
+.rateOurWork {
+  margin-top: 12px;
+}
+
+.favBrandCar {
+  margin-bottom: 0%;
+}
+
+.comments {
+  margin-bottom: 10px;
+}
+
+.factsAboutYou {
+  padding-bottom: 4px;
+}
```

Comment:
you can create 2 classes: 1, for each element that should have magrin 10, 2. for each element that should have magrin 20, that's all